packages/date-fns/src/index.ts

```diff
@@ -413,6 +414,11 @@ class DateFnsUtils implements IUtils<Date> {
 
     return years;
   };
+
+  public fromNow = (date: Date) => {
```

Comment:
@menna-mostafa recheck the actual format "days/ mins..etc?" handle "ago localization" or only time format